src/ResourceManager/Compute/Commands.Compute/Generated/Disk/DiskDeleteMethod.cs

```diff
@@ -79,8 +79,7 @@ protected void ExecuteDiskDeleteMethod(object[] invokeMethodInputParameters)
             string resourceGroupName = (string)ParseParameter(invokeMethodInputParameters[0]);
             string diskName = (string)ParseParameter(invokeMethodInputParameters[1]);
 
-            var result = DisksClient.Delete(resourceGroupName, diskName);
-            WriteObject(result);
+            DisksClient.Delete(resourceGroupName, diskName);
```

Comment:
what is this actually doing, with the method below calling async methods?  Which is actually used in cmdlet execution?